cranelift-codegen/src/simple_preopt.rs

```diff
@@ -450,76 +452,85 @@ fn do_divrem_transformation(divrem_info: &DivRemByConstInfo, pos: &mut FuncCurso
     }
 }
 
+#[inline]
+fn resolve_imm64_value(dfg: &DataFlowGraph, value: Value) -> Option<immediates::Imm64> {
+    if let ValueDef::Result(iconst_inst, _) = dfg.value_def(value) {
```

Comment:
Nit: "inst" or "candidate_inst" would be more reasonable since we don't yet know it's an Iconst.